lib/active_merchant/billing/gateways/cyber_source_rest.rb

```diff
@@ -324,9 +434,9 @@ def sign_payload(payload)
       def auth_headers(action, post, http_method = 'post')
         digest = "SHA-256=#{Digest::SHA256.base64digest(post.to_json)}" if post.present?
         date = Time.now.httpdate
-
+        accept = /payments/.match?(action) || /credits/.match?(action) ? 'application/hal+json;charset=utf-8' : 'application/json;charset=utf-8'
         {
-          'Accept' => 'application/hal+json;charset=utf-8',
+          'Accept' => accept,
```

Comment:
💬 Comment:

We should revisit this at the end, seem very improbable that an "Accept" header has an impact on the actual request.

Reply:
It's weird, but modifying the "accept" affects the sandbox response for these "store" calls.

Reply:
Maybe use 'Accept' => 'application/hal+json, application/json;charset=utf-8', which accepts both media types.